addons/account/models/chart_template.py

```diff
@@ -306,6 +306,19 @@ def tax_template_changed(tax, template):
                                 'record': existing_account,
                                 'noupdate': True,
                             }])
+                elif model_name == 'account.group':
+                    account_group = self.ref(xmlid, raise_if_not_found=False)
+                    if not account_group or (account_group and account_group.code_prefix_start == values['code_prefix_start']):
```

Comment:
?
```suggestion
                    if not account_group or (account_group and account_group.code_prefix_start != values['code_prefix_start']):
```

Reply:
done

Reply:
What about the main comment of the review?